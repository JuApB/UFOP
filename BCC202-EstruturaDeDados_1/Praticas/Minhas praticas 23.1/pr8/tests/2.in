2
21
MariaFernanda 18 1
Analu 31 1
Heloisa 32 2
Anthony 69 2
Pedro 75 3
Rafaela 97 2
JoaoMiguel 50 1
Gabriel 16 1
ArthurHenrique 16 2
Ryan 68 3
Sophia 12 1
Liz 48 2
Gabrielly 72 2
Aurora 55 1
AnaLivia 35 3
Luiza 46 3
Pietro 37 3
Eduarda 92 1
Francisco 28 2
Caio 29 1
AnaSophia 50 3
2
Luna 46 2
Laura 54 3
