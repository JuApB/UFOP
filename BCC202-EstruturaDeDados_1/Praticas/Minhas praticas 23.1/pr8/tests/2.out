Pedro 75 3
Ryan 68 3
AnaLivia 35 3
Luiza 46 3
Laura 54 3
Rafaela 97 2
Heloisa 32 2
Pietro 37 3
Francisco 28 2
Analu 31 1
AnaSophia 50 3
Liz 48 2
Gabrielly 72 2
Aurora 55 1
JoaoMiguel 50 1
Gabriel 16 1
Anthony 69 2
Eduarda 92 1
ArthurHenrique 16 2
Caio 29 1
MariaFernanda 18 1
Sophia 12 1
Luna 46 2
